archweb.wsgi

```diff
@@ -5,7 +5,7 @@ import site
 
 base_path = "/srv/http/archweb"
 
-site.addsitedir('/srv/http/archweb-env/lib/python2.7/site-packages')
+site.addsitedir('/srv/http/archweb-env/lib/python3.6/site-packages')
```

Comment:
PR title says Python 3.7 but this says 3.6

Reply:
Yeah this will not work.